### PR TITLE
Adding book and author loaders

### PR DESCRIPTION
**Pull request by katiacammisa**
# Description

Adding book and author loaders

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

Please describe the tests that you ran to verify your changes.

- [ ] Automation testing (Unit or integration tests)
- [ ] Manual testing (Postman)

# Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] My pull request is up to date with develop

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -122,6 +122,10 @@
 			<version>4.12</version>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```
Comment (brianfroschauer) [inline pom.xml:125]: Esta dependencia se agrega por algún motivo?

File: src/main/java/com/austral/bookin/loader/BookLoader.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                "Inglés",
+                new Date(101, Calendar.DECEMBER, 1));
+
+        final Book book6 = new Book(
+                "Harry Potter and the Philosopher's Stone",
+                "Fantasía",
+                "Inglés",
+                new Date(97, Calendar.DECEMBER, 26));
+
+        bookService.save(book1, new MockMultipartFile("huckleberry.jpg", new FileInputStream(new File("src/main/java/com/austral/bookin/util/books/huckleberry.jpg"))));
```
Comment (brianfroschauer) [inline src/main/java/com/austral/bookin/loader/BookLoader.java:66]: las fotos no deberían estar en /resources? yo las pondría en /resources/assets/books/.**

**Review by brianfroschauer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (katiacammisa) [reply pom.xml:125]: Si, la dependencia es para poder usar el Mock para convertir el File a MultipartFile

File: src/main/java/com/austral/bookin/loader/BookLoader.java (hunk 2 above)
Comment (katiacammisa) [reply src/main/java/com/austral/bookin/loader/BookLoader.java:66]: Dale, ahora las cambio

**Pull request merged**